etc/build/Makefile

```diff
@@ -12,7 +12,13 @@ $(shell mkdir -p $(tmplocation))
 
 bblfsh-sdk := $(shell command -v bblfsh-sdk 2> /dev/null)
 bblfsh-tools := $(shell command -v bblfsh-tools 2> /dev/null)
-ifdef bblfsh-tools
+in-container := $(shell echo $$ENVIRONMENT)
+
+# TODO(bzz) ideal: `go run` ONLY inside container, started from non-linux host
+ifdef bblfsh-tools # with Golang
```

Comment:
At first I also thought so, but it looks like this was here before, together with `command -v` for a reason: to avoid running this in native build container WITHOUT go.